Lib/test/support/interpreters.py

```diff
@@ -0,0 +1,175 @@
+"""Subinterpreters High Level Module."""
+
+import _xxsubinterpreters as _interpreters
```

Comment:
The simplest approach with some of the pieces is to expose them directly from the low-level module:

```suggestion
import _xxsubinterpreters as _interpreters
# aliases:
from _xxsubinterpreters import (
    ChannelError, ChannelNotFoundError,
    ChannelEmptyError, ChannelNotEmptyError, NotReceivedError,
    is_shareable,
)
```